infrastructure/saat.tfvars

```diff
@@ -0,0 +1 @@
+idam_api_url = "http://betadevbccidamapplb.reform.hmcts.net/""
```

Comment:
can we update the default one please?
CMC is happy for this to be changed till the 25th

Reply:
Waiting for Divorce to confirm

Reply:
@timja-kainos + @nitinprabhu Divorce doesn't look to be using SIDAM in any env, including SAAT:
https://github.com/hmcts/div-petitioner-frontend/blob/master/infrastructure/saat.tfvars#L17
https://github.com/hmcts/div-evidence-management-client-api/blob/master/infrastructure/saat.tfvars#L2
https://github.com/hmcts/div-case-progression-service/blob/master/infrastructure/saat.tfvars#L7

Reply:
However it looks like you've got an extra `"` in there